ui/app/AppLayouts/Profile/Sections/BackupSeedModal.qml

```diff
@@ -0,0 +1,117 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import "../../../../imports"
+import "../../../../shared"
+
+ModalPopup {
+    id: popup
+
+    title: qsTr("Write down your seed phrase")
+
+    Item {
+
+      id: seed
+      anchors.left: parent.left
+      anchors.right: parent.right
+      width: parent.width
+      height: children[0].height
+
+      Component {
+        id: wordCmp
+
+        Item {
+          height: 40
+          anchors.left: parent.left
+          anchors.leftMargin: Theme.bigPadding
+
+          TextEdit {
+            text: word
+            font.pixelSize: 15
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: Theme.smallPadding
+            selectByMouse: true
+            readOnly: true
+          }
+        }
+      }
+
+
+      Rectangle {
+        anchors.top: parent.top
+        anchors.topMargin: Theme.padding
+        height: 250
+        width: 350
+        border.width: 1
+        border.color: Theme.grey
+        radius: Theme.radius
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        Rectangle {
+          id: mnemonicLeft
+          width: 175
+          anchors.top: parent.top
+          anchors.left: parent.left
+          anchors.bottom: parent.bottom
+          color: "transparent"
+
+          ListView {
+              property int counter: 0
+              anchors.fill: parent
+              model: profileModel.mnemonicLeft
+              delegate: wordCmp
+          }
+        }
+
+        Rectangle {
+          width: 1
+          anchors.top: parent.top
+          anchors.bottom: parent.bottom
+          color: Theme.grey
+          anchors.left: parent.left
+          anchors.leftMargin: 174
+        }
```

Comment:
My take on creating a vertical separator...

Reply:
I think we already have a separator component (it's used in the ModalPopup component at least)